src/server/node_manager.c

```diff
@@ -6760,7 +6760,10 @@ build_execvnode(job *pjob, char *nds)
 		pc = parse_plus_spec(NULL, &rc);
 	}
 
-	if (rc)
+        /* if the number of nodes defined above in ndarray is not equal */
+        /* to the number of nodes identified, then skip the below loop */
```

Comment:
Use the PBS standard for multi-line comments: A /* start, followed by *'s down the left hand side, ending with a */ on a line by itself.